infra/gcp/bash/lib.sh

```diff
@@ -452,9 +479,9 @@ function empower_artifact_auditor() {
 # subscription getting its messages from the GCR topic "gcr", where changes to
 # GCR are posted).
 # $1: The GCP project
-function empower_artifact_auditor_invoker() {
+function empower_image_auditor_invoker() {
```

Comment:
I would double check the naming in `ensure-env-cip-auditor.sh` as well while you're at it